Initial/draft fix for #33 Chrome's displaying issue

danimesq opened the pull request:
This is all that I know for make it display like a charm in Google
Chrome and other webkit browsers. You can complement this draft fix.

Commit:
Initial/draft fix for #33 Chrome's displaying issue

This is all that I know for make it display like a charm in Google
Chrome and other webkit browsers. You can complement this draft fix.

nt1m commented on css/style.css:
```diff
@@ -303,6 +303,7 @@ body {
 	background-image: url(../img/volume_mute.svg);
 }
 #volume-range {
+	-webkit-appearance:none;
```
Can you add a space after every ":" character ?

nt1m commented on css/style.css:
```diff
@@ -324,6 +326,25 @@ body {
+	width: 16px;
+	border: none;
+}
+#volume-range::-webkit-slider-progress {
```
Does this even work ? I thought it wasn't supported by webkit

DaniellMesquita commented:
Yes, its supported, but I don't know how do this completly. I've only made the track white and the thumb rounded+cyan-colored. This is only a draft for a future complete work. 
As I know, the one thing that webkit don't support is stylising the progress.

Commit:
Added a space after every ":" character in rangebar

DaniellMesquita commented:
I've added a space every ":" character in rangebar's CSS. My work is finished in this #34 pull request, unless you need more tune ups here.

nt1m commented:
@DaniellMesquita That means the `#volume-range::-webkit-slider-progress` part doesn't work.

nt1m commented:
In my opinion, it's better to just remove that rule, until webkit provides an alternative to ::-moz-range-progress.

Commit:
Removed a CSS until webkit provides an alternative

Removed  #volume-range::-webkit-slider-progress CSS rule until webkit
provides an alternative to ::-moz-range-progress. Thanks to @nt1m aka
Tim Nguyen for the tip.

DaniellMesquita commented:
I've removed that errouneous CSS rule that don't exists yet. Thanks for the tip. I'll open a issue  in webkit project requesting an alternative to `::-moz-range-progress`. My work has finished in this initial/draft #34 pull request, unless you need more tune ups here.

DaniellMesquita commented:
I've removed that errouneous CSS rule I've that don't exists yet. Thanks for the tip. I'll open a issue  in webkit project requesting an alternative to `::-moz-range-progress`. My work has finished in this initial/draft #34 pull request, unless you need more tune ups here.

nt1m commented:
Thanks !
